knexfile.js

```diff
@@ -0,0 +1,14 @@
+// Update with your config settings.
+
+module.exports = {
+  development: {
+    client: "sqlite3",
+    connection: {
+      filename: "./data/car-dealer.db3"
+    },
+    migrations: {
+      directory: "./data/migrations"
+    },
```

Comment:
when you do seed tables in the future remember to add the directory here as well